src/FileNames.h

```diff
@@ -240,6 +240,9 @@ namespace FileNames
    AUDACITY_DLL_API
    //! Give enough of the path to identify the device.  (On Windows, drive letter plus ':')
    wxString AbbreviatePath(const wxFileName &fileName);
+
+   //! Return path separator char for current filesystem.
+   AUDACITY_DLL_API wxChar GetPathSeparator();
```

Comment:
This might be just a constexpr variable.

Or even just wxPATH_SEPARATOR which is acceptable (belonging to wxBASE)